docs/config.rst

```diff
@@ -1677,6 +1679,36 @@ Valid preset color variables are:
 
    See also: :attr:`LP_ENABLE_BATT`.
 
+.. attribute:: LP_COLOR_CMAKE_BUILD
+   :type: string
+   :value: $MAGENTA
+
+   Color used to display the build type in the CMake segment.
+
+   See :attr:`LP_ENABLE_CMAKE`.
+
+   .. versionadded:: 2.2
+
+.. attribute:: LP_COLOR_CMAKE_C
+   :type: string
+   :value: $MAGENTA
+
+   Color used to display the C compiler in the CMake segment.
+
+   See :attr:`LP_ENABLE_CMAKE`.
+
+   .. versionadded:: 2.2
+
+.. attribute:: LP_COLOR_CMAKE_CXX
```

Comment:
Maybe we should consider making an array for cmake colors. That is tough though as long as we can't use associative arrays / dictionaries.

Reply:
It's been several time the associative arrays may have been a cleaner option. Should we consider bumping shell version requirements?

Reply:
Considering it is a good idea. My gut reaction is that we should release v2.2 first, since it would be a breaking change and would fit better in a v3.0. But we should talk about it.

Reply:
So, should we merge this PR and aim for the release?